_docs/software_support_lifecycle.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Software Support Lifecycle
+date: 2020-12-09
+summary: Vox Pupuli policy on supporting upstream software
+---
+
+## Software Support Lifecycle
+
+As a part of Vox Pupuli's mission to provide a home for modules that still have a userbase and need development, we need to define a clear support strategy for what operating systems, dependencies, and upstream software we will support and for how long those will be supported.
+
+As we operate entirely on the graces of our community members being willing to voluntarily contribute to our modules, we also need to be respective of their time and resources and not add undue stress in order to support software that upstream creators no longer support.
+
+To account for this, we no longer support any software or operating system after the date that the maintainers have marked said software or operating system as End of Life. However, this does not mean we will immediately remove code that allows for the module to run on End of Life'd software or operating systems, but the community will no longer be expected to provide support to users for that software.
+
+There are exceptions and the full policy is detailed in the next section.
+
+### Support Lifecycle
+
+#### Operating Systems
+An Operating System is no longer considered supported once it reaches an End of Life date determined by the upstream maintainer* of the operating system. To ensure our users have enough time to adjust, we will take the following steps to ensure as seamless of a transition as possible:
```

Comment:
there are some edge cases. for example debian has still "support" for debian 8, but that's very very limited. In the past we considered that as not-supported. Same applies for RedHat 5/6 where you can only get support if you buy it from redhat. I also consider those distros as not supported anymore

Reply:
I think we do need to decide ahead of time which versions we will support.

For Debian, do we want to support until EOL or until End of LTS. This one is confusing as other LTS operating systems, like RHEL and Ubuntu, have EOL and end of LTS as the same thing and post LTS support as "Extended Support".

I think for RHEL, I'd like to end our support on the EOL date, not the End of Extended support. That only affects paying RHEL customers and we aren't in the "business" of supporting Enterprises. We build for any user and as such our focus needs to be on what's best for the community as a whole, not the Enterprise users only.